src/app/components/progress-indicator/index.html

```diff
@@ -1,20 +1,266 @@
-<sky-demo-page
+<sky-tabbed-demo-page
   pageTitle="Progress Indicator"
-  navTitle="Progress indicator"
-  skyuxModule="SkyProgressIndicatorModule"
-  packageName="@skyux/progress-indicator"
 >
 
   <sky-demo-page-summary>
     <p>
-      The progress indicator component visually represents progress through a series of sequential steps toward a final goal. Active progress indicators represent tasks where user actions are broken into discrete steps, and passive progress indicators represent tasks that concern users but where they do not directly act upon the steps.
+      The progress indicator component shows progress through steps toward a goal. The passive progress indicator represents steps that take place outside of the user’s control. The active progress indicator walks users through a multi-step process, either in the waterfall page (below) or in the <a routerLink="/components/wizard">Wizard</a> component.
     </p>
-    <p>
-      Active progress indicators can be modal wizards or waterfall pages. For <a routerLink="/components/wizard">modal wizards</a>, place progress indicator components within modal components to guide users through tasks that require a series of related decisions. For waterfall pages, place progress indicator components on pages where records or system settings are in an incomplete state until users complete steps to activate them.
-    </p>
- </sky-demo-page-summary>
+  </sky-demo-page-summary>
+    
+<div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+        <sky-progress-indicator-demo>
+        </sky-progress-indicator-demo>
+    </sky-demo-page-example>
+</div>
+    
+<sky-tabset id="content-tabs">
+	
+	<sky-tab
+		tabHeading="Waterfall indicator design">
+        
+	
+    
+    <sky-demo-page-content sectionHeading="Usage">
+	<h3 class="sky-text-success">Use when</h3>
+    
+        <sky-docs-visual-example>
```

Comment:
Is it worth showing this in a full page context? Seems like we often see examples where the title is top left and the waterfall is centered which looks very weird.

Reply:
I added an example of a full page context in the Layout section on the page. Think that's good enough?

Reply:
yup

Reply:
@Blackbaud-AdamFunderburk, it looks like the first image in the Don't use when section was also updated to a full-page context and is now much larger. None of the other Don't use when images (or Use when images) were updated. Was that change to that one Don't use when image intentional?

Reply:
Resolving as Adam and I talked about this and the first Use when image was updated...